src/unix/core.c

```diff
@@ -198,6 +198,19 @@ void uv__make_close_pending(uv_handle_t* handle) {
   handle->loop->closing_handles = handle;
 }
 
+int uv__getiovmax(void) {
+#if defined(IOV_MAX)
+  return IOV_MAX;
+#elif defined(_SC_IOV_MAX)
+  static int iovmax = -1;
+  if (iovmax == -1)
+    iovmax = sysconf(_SC_IOV_MAX);
```

Comment:
`a method for applications to determine the current value of a configurable system limit or option variable`

By determine, I don't think it means "can change". I think it means "to read its value".